wallet/src/main/java/io/neow3j/wallet/Account.java

```diff
@@ -245,64 +271,55 @@ public NEP6Account toNEP6Account() {
      *                           transactions.
      * @return the multi-sig account builder;
      */
-    public static Builder fromMultiSigKeys(List<ECPublicKey> publicKeys, int signatureThreshold) {
```

Comment:
This comment has nothing to do with your changes. I just realized that this method's name is not intuitive. Let's rename it to `createMultiSigAccount()`.